### ISSUE TITLE
My style bindings are disappearing after production build

### ISSUE THREAD

**Issue by esahione** [vuejs-templates/webpack#53]
My style bindings are not working after I build the code for production (I'm using the Webpack skeleton initially generated by vue-cli).

The code is for a simple parallax component - you can find it here: https://gist.github.com/esahione/6a7dc75f1dec971c4c06

Here's my Webpack production config file: https://gist.github.com/esahione/c8d75d51ff7e59b10d5a
And here's the Webpack development configuration file: https://gist.github.com/esahione/c305121816040f849547

The code should be self explanatory. It works flawlessly on development, but when I try to build it for production it stops working. In fact, when I look at the code through the browser there is no style tag.

Is this a bug? How can I fix it?

**Comment by esahione** [comment]
So... I disabled the extract-text-webpack-plugin and now it's working.

**Comment by Astray-git** [comment]
I have the same issue if build on Debian (build config works fine on OSX and Windows).
May be related to #41 ?

**Comment by chrisvfritz** [comment]
Sounds like it is possibly related. @esahione @Astray-git I'd [open an issue in extract-text-webpack-plugin](https://github.com/webpack/extract-text-webpack-plugin/issues/new) detailing your issue. If you find a solution to your problem, please let us know back here!

**Comment by chrisvfritz** [comment]
I'm pretty convinced this is indeed the same issue as #41, so closing as duplicate for now.

**Comment by Astray-git** [comment]
Well, in my situation, the problem is not related to extract-text-webpack-plugin.

I find out the `minifyCSS` option for html-minifier will remove some `:style` biding code. (tested on  Ubuntu 15.10)
‘minifyCSS' is set to true by default in vue-html-loader:
https://github.com/vuejs/vue-html-loader/blob/master/index.js#L78

**Comment by yyx990803** [comment]
@Astray-git what exactly is being removed? Example? Repro?

**Comment by Astray-git** [comment]
code:
```html
<th
  :style="[
    column.styles,
    { width: column.width }
  ]">
```
build without `miinifyCSS`:
```html
<th :style="[\n            column.styles,\n            { width: column.width }\n          ]">
```

build with `minifyCSS`: 
```html
<th :style="">
```

**Comment by chrisvfritz** [comment]
@Astray-git I unfortunately wasn't able to reproduce the problem with that example. :confused: Could you link to an example project demonstrating the issue?

**Comment by Astray-git** [comment]
@chrisvfritz Try this [hello.vue](https://gist.github.com/Astray-git/f76bc99cede942bd381fb454effa9099) .
 I can reproduce the bug in a fresh webpack-template project.

**Comment by chrisvfritz** [comment]
@Astray-git Thanks for the link! That helped a lot. :smiley: From [the Vue docs](http://vuejs.org/guide/class-and-style.html#Array-Syntax-1):

> The Array syntax for `v-bind:style` allows you to apply multiple style __objects__ to the same element:

That means `column.styles` should be an object rather than a string. For example:

``` js
column: {
  width: '10px',
  styles: {
    textAlign: 'center'
  }
}
```

Is your issue resolved after making that change?

**Comment by Astray-git** [comment]
@chrisvfritz Sorry, that's a mistake in my test code.
with a correct style object, the style biding template is still built to `:style=''`

**Comment by yyx990803** [comment]
This seems like an `html-minifier` bug, but those options are not needed for Vue templates anyways. Fixed in `vue-html-loader@1.2.1`.
